src/migratus/cli.clj

```diff
@@ -95,17 +101,99 @@
 
       :else (no-match-message args summary))))
 
+(defn util-date-to-local-datetime [util-date]
+  (when (some? util-date)
+    (let [instant (.toInstant util-date)
+          zone-id (ZoneId/systemDefault)
+          local-datetime (.atZone instant zone-id)]
+      local-datetime)))
+
+(defn formatted-date [date]
```

Comment:
If you format the date as string and print the string, then it's better to use date time formatters https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME